### PR TITLE
feat(docs): add POC for use-cases - accordion example

### PR DESCRIPTION
**Pull request by ludmilanesvitiy**
# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

File: docs/spec/accordion/accordion.examples.group-open.use-case.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Goal**: Show example of event listener, which listen all actions with accordion items
+
+Main success scenario:
+----------------------
+1. User open Accordion demo page
+2. User click on Group opening event sub-menu
+3. User able to click on each item of 3
+4. After click on item without event listener, item is opened and in the browser console - nothing happens
+5. After click on item with event listener, item is opened and in the browser console - should be message "Accordion has been opened"
+
```
Comment (OlegDubrovskyi) [inline docs/spec/accordion/accordion.examples.group-open.use-case.md:16]: Need to remove extra line

File: docs/spec/accordion/accordion.examples.group-open.use-case.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Extensions:
+-----------
+4a. If item is opened, after click on item without event listener, item is closed and in the browser console - nothing happens
+5a. If item is opened, after click on item with event listener, item is closed and in the browser console - should be message "Accordion has been closed"
+
```
Comment (OlegDubrovskyi) [inline docs/spec/accordion/accordion.examples.group-open.use-case.md:21]: Need to remove extra line

File: docs/spec/accordion/accordion.examples.group-open.use-case.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Variations:
+-----------
+2*. User scroll to Group opening event sub-menu
+
```
Comment (OlegDubrovskyi) [inline docs/spec/accordion/accordion.examples.group-open.use-case.md]: Need to remove extra lines

File: docs/spec/accordion/accordion.examples.initially-opened.use-case.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Main success scenario:
+----------------------
+1. User open Accordion demo page
+2. User click on Initially opened sub-menu
+3. User able to click on each item of 3
+4. Item with name "Initially expanded" should be opened by default and content inside should be visible
+5. When user click on this item, it should be closed
+6. When user reload page, then initially expanded item should be opened again
+
```
Comment (OlegDubrovskyi) [inline docs/spec/accordion/accordion.examples.initially-opened.use-case.md:17]: Need to remove extra line

**Review by OlegDubrovskyi — DISMISSED** [review]

File: docs/spec/accordion/accordion.examples.basic.use-case.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+0.1: Accordion basic example
+============================
+**Primary Actor**: User
+
+**Scope**: Ngx-bootstrap DEMO / BS version 4/ v.3.0.1
```
Comment (valorkin) [inline docs/spec/accordion/accordion.examples.basic.use-case.md]: Remove ngxbs version please
BS version 3&4

**Review by valorkin — CHANGES_REQUESTED** [review]
Same for other versions
When and how svgs are rendered?

**Comment by ludmilanesvitiy** [comment]
fixed versions for all files
svg generated in online editor https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbkFbQ2hyaXN0bWFzXSAtLT58R2V0IG1vbmV5fCBCKEdvIHNob3BwaW5nKVxuQiAtLT4gQ3tMZXQgbWUgdGhpbmt9XG5DIC0tPnxPbmV8IERbTGFwdG9wXVxuQyAtLT58VHdvfCBFW2lQaG9uZV1cbkMgLS0-fFRocmVlfCBGW2ZhOmZhLWNhciBDYXJdXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9fQ
But besides use-case, for generating svg we need to support Mermaid configuration, it's a double work. That's why I don't recommend using it now. This is just example how it could looks like

**Review by valorkin — APPROVED** [review]

**Review by valorkin — DISMISSED** [review]

**Review by valorkin — DISMISSED** [review]

**Review by valorkin — APPROVED** [review]

**Pull request merged**